lib/CPAN/Audit.pm

```diff
@@ -199,6 +199,8 @@ sub command_installed {
 		$dists->{ $dep->{dist} } = $dep->{version};
 	}
 
+	map { $_ = "==$_" } values %$dists;
```

Comment:
Why not do it 2 lines above where that value is set in the first place?

Reply:
We're not going to do this a map in void context, but if we need to do this it will get done somehow.

Reply:
Well, why not do it 2 lines above?

If I read the code at hand correctly: The author was in doubt if `CPAN::Audit::Installed::find()` would always manage to return a complete list of dependencies and decided to base it on the result of the evaluation of `$Module::CoreList::version{$]}` instead (which resides in the `$dists` parameter and is complemented/updated by the result of `CPAN::Audit::Installed::find()`).

As I am not in the position to assess if the result of `CPAN::Audit::Installed::find()` is always comprehensive (in that case `sub command()` should not evaluate `$Module::CoreList::version{$]}` at all for command `installed`) and know better than the author, I decided to stick with the defensive approach and update `$dists` instead.

Reply:
The summary of that is that `foreach` skips some hash elements, so some hash elements would be missed.